sdk/storage/storage-blob/src/utils/utils.common.ts

```diff
@@ -96,34 +96,43 @@ function getProxyUriFromDevConnString(connectionString: string): string {
   return proxyUri;
 }
 
+function getValueInConnString(
+  connectionString: string,
+  argument:
+    | "BlobEndpoint"
+    | "AccountName"
+    | "AccountKey"
+    | "DefaultEndpointsProtocol"
+    | "EndpointSuffix"
+    | "SharedAccessSignature"
+) {
+  const matchCredentials = connectionString.split(";");
+  for (const element of matchCredentials) {
+    if (element.trim().startsWith(argument)) {
+      return element.trim().match(argument + "=(.*)")![1];
+    }
+  }
+  return "";
+}
+
 /**
  * Extracts the parts of an Azure Storage account connection string.
  *
  * @export
  * @param {string} connectionString Connection string.
- * @returns {{ kind: "AccountConnString" | "SASConnString", url: string, [key: string]: any }} String key value pairs of the storage account's url and credentials.
+ * @returns {ConnString}  String key value pairs of the storage account's url and credentials.
```

Comment:
nit: not related to this PR but full name `ConnectionString` sounds better

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/5330